examples/stable-diffusion/README.md

```diff
@@ -584,3 +584,65 @@ python image_to_video_generation.py \
 
 > For improved performance of the image-to-video pipeline on Gaudi, it is recommended to configure the environment
 > by setting PT_HPU_MAX_COMPOUND_OP_SIZE to 1.
+
+# Diffusers Inference techniques
```

Comment:
I wouldn't set it here with this section name.
Rather `## Additional inference techniques` that could be added right before https://github.com/yuanwu2017/optimum-habana/tree/tech/examples/stable-diffusion#stable-video-diffusion-examples.